packages/host/tests/unit/index-writer-test.ts

```diff
@@ -48,14 +48,6 @@ module('Unit | index-writer', function (hooks) {
     });
   });
 
-  test('only invalidates latest version of content', async function (assert) {
```

Comment:
this becomes moot since we are now breaking out the working version and the production version into separate tables